configuration/external_parameters.rst

```diff
@@ -494,6 +494,46 @@ Symfony provides the following env var processors:
     .. versionadded:: 4.2
         The ``key`` processor was introduced in Symfony 4.2.
 
+``env(default:fallback_param:BAR)``
+    Retrieves the value of the parameter ``fallback_param`` when the of the ``BAR`` env var is not available:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # config/services.yaml
+            parameters:
+                private_key: '%env(default:raw_key:file:PRIVATE_KEY)%'
+                raw_key: '%env(PRIVATE_KEY)%'
```

Comment:
I wonder if we could put a hardcoded path/value for `raw_key` to make the example easier to understand. Using an env var as the default value of the env var value of another value makes my head explode 💥